packages/react-components/package.json

```diff
@@ -58,9 +59,10 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "sass": "^1.49.9",
+    "storybook": "^7.0.20",
     "storybook-addon-themes": "^6.1.0",
     "typescript": "^4.5.4",
-    "vite": "^2.8.0",
+    "vite": "^3.0.0",
     "vite-plugin-dts": "^0.9.10",
```

Comment:
I'm wondering if maybe we can update `vite-plugin-dts` as well. Current version is `2.3.0`.